### ISSUE TITLE
Increase Tagify maxItems setting

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#580]
**What are you trying to do?**
We've run into an issue where our site having a large amount of tags is leaving users unable to select what they want using the drop down as it only returns the first 10 tags as per the Tagify `dropdown.maxItems` setting

For example, here they want to search for and select "White", but the dropdown returns 10 variations of it first

<img width="288" alt="Screen Shot 2021-09-20 at 2 00 29 pm" src="https://user-images.githubusercontent.com/21274925/133957391-a1d86639-cf35-4806-8161-0234c670aae0.png">

We are aware that just clicking off the field still selects the tag, but users seem to getting confused by this.
Ideally the variations would also be separate tags, but as we're pulling the tag data from a third party API it's hard to keep that in check

**What's your proposed solution?**
Increase the default maxItems setting, or add an option for it to be increased through field settings (unsure how viable this is)

**Comment by engram-design** [comment]
Added new events for you for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.17",
  "...": "..."
}
```

Then run `composer update`. You can modify the options for the [tagify](https://github.com/yairEO/tagify#settings) JS plugin via [events](https://verbb.io/craft-plugins/formie/docs/developers/javascript-api#tag-fields)

**Comment by d-karstens** [comment]
Thank you!
Just tested and that works perfect for the issue we were having 👍

**Comment by engram-design** [comment]
Fixed in [1.4.18](https://github.com/verbb/formie/releases/tag/1.4.18)
